### PR TITLE
Refactor/thread card

### PR DESCRIPTION
**Pull request by RSNara**
This pull request splits the `ThreadCard` into several different smaller components. They are domain specific for the time being. There aren't too many UI components yet, but we can worry about them later

### CODE REVIEW

**Comment by RSNara** [comment]
Rebasing.

**Comment by RSNara** [comment]
I love this.

File: package.json [hunk 1]
```diff
@@ -71,11 +71,9 @@
     "basscss-typography": "^3.0.3",
     "chai": "^3.4.0",
     "classnames": "^2.2.3",
-    "commitizen": "^2.8.1",
     "cross-env": "^1.0.7",
     "css-loader": "^0.23.0",
     "cssnano": "^3.5.2",
-    "cz-conventional-changelog": "^1.1.6",
```
Comment (JaKXz) [inline package.json:78]: good catch!

**Comment by JaKXz** [comment]
PRime squash-merge candidate :) architecture LGTM!

**Pull request merged**